src/controller/Controller.java

```diff
@@ -3,30 +3,72 @@
 import view.View;
 import model.*;
 
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.logging.*;
+
 public class Controller {
 	
 	private View view;
-	private Password assignedPassword;
-	private Password enteredPassword;
+	private Password assignedPassword; // password system gives them
+	private Password enteredPassword; // what they give us
 	
 	public Controller(){
 		//TODO: randomly assign a password here
+		assignedPassword = new Password(/*take in assigned values */ );
+		
+		enteredPassword = new Password();
 		view = new View(this);
 		view.setVisible(true);
-		
 	}
 	
 	public void handleSubmit(){
-		//TODO: make this do things
+		boolean success = assignedPassword == enteredPassword;
+		
+		// TODO: change the state based off of success
+		
+		String event = success ? "success" : "failure";
+		// log success or failure event
+		logEvent("user,scheme,login," + event);	
 	}
 	
-	public void handleTextEnter(){
-		//TODO: entered text to be added to entered password
+	
+	public void handleTextEnter(String textpw){
+		enteredPassword.setRandomWord(textpw);
+	}
+	
+	// Input: A string to be logged in the specified format
+	// FORMAT: user,scheme,mode,event
+	public static void logEvent(String logMe) {
+		Logger logger = Logger.getLogger("MyLog");  
+	    FileHandler fh;  
+ 
+	    try {  
+	        // This block configure the logger with handler and formatter  
+	        fh = new FileHandler("C:/Users/Chantal/Documents/hcilog/temp.log");  
```

Comment:
Also, if it is relative then log files will be added to github, Making it easier to ensure that we have all of the logs in the end :)